src/lightning/fabric/strategies/fsdp.py

```diff
@@ -341,8 +343,8 @@ def module_init_context(self, empty_init: Optional[bool] = None) -> ContextManag
             stack.enter_context(torch.device("meta"))
         elif _TORCH_GREATER_EQUAL_1_13:
             stack.enter_context(_EmptyInit(enabled=bool(empty_init)))
```

Comment:
why isn't it applied in other places here?

Reply:
`torch.device` wont raise an exception but I missed this `_EmptyInit`

Reply:
Can you include it in #18734?